app/src/main/java/ai/elimu/appstore/service/GetAppListByCollectionIdService.java

```diff
@@ -11,8 +11,8 @@
  */
 public interface GetAppListByCollectionIdService {
 
-    @GET("project/app-collections/{id}/applications")
-    Call<ResponseBody> getApplicationListByCollectionId(@Path("id") long id,
+    @GET("project/app-collections/{appCollectionId}/applications")
+    Call<ResponseBody> getApplicationListByCollectionId(@Path("appCollectionId") Long appCollectionId,
```

Comment:
Why do you have to change from long to Long?

Reply:
Just to match type used for the Java class represented by the JSON request/response: https://github.com/elimu-ai/model/blob/master/src/main/java/ai/elimu/model/gson/BaseEntityGson.java